weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/schemaUtils.ts

```diff
@@ -141,7 +175,7 @@ export const extractSourceSchema = (calls: CallData[]): SchemaField[] => {
   });
 
   return allFields
-    .filter(field => field.name !== 'inputs.self')
+    .filter(field => !field.name.startsWith('inputs.self'))
```

Comment:
is this because we now resolve the ref and there can be sub-fields?

Reply:
Precisely!